src/main/java/com/fcgl/madrid/points/service/UserTrophyService.java

```diff
@@ -19,4 +33,38 @@ public UserTrophyService(UserTrophyRepository userTrophyRepository) {
     public List<UserTrophy> findAll() {
         return userTrophyRepository.findAll();
     }
+
+    public List<UserTrophy>getUserTrophiesById(Long userId) {
+      return userTrophyRepository.getUserTrophiesById(userId);
+    }
+
+    // @CircuitBreaker(name = "backendA", fallbackMethod = "fallback")
+    // public ResponseEntity<InternalStatus>
+
+    /**
+    *@param
+    *@return
+    */
+    private ResponseEntity<InternalStatus> handleParamException(TransactionSystemException e) {
```

Comment:
We want to stay away from having duplicate functions. Since this function as well as `getInternalStatusResponseEntity` is used twice I would recommend to add another class... Perhaps `ExceptionUtil` and add `handleParamException` and `getInternalStatusResponseEntity` there as public functions. Then import that class and call the necessary functions